pkg/artifactory/repositories.go

```diff
@@ -700,22 +708,23 @@ func getDefaultDeploymentRepo(d *ResourceData) string {
 	return value
 }
 
-func unpackBaseVirtRepo(s *schema.ResourceData) VirtualRepositoryBaseParams {
+func unpackBaseVirtRepo(s *schema.ResourceData, packageType string) VirtualRepositoryBaseParams {
 	d := &ResourceData{s}
 
 	return VirtualRepositoryBaseParams{
 		Key:    d.getString("key", false),
 		Rclass: "virtual",
 		//must be set independently
-		PackageType:     "invalid",
+		PackageType:     packageType,
 		IncludesPattern: d.getString("includes_pattern", false),
 		ExcludesPattern: d.getString("excludes_pattern", false),
 		RepoLayoutRef:   d.getString("repo_layout_ref", false),
 		ArtifactoryRequestsCanRetrieveRemoteArtifacts: d.getBool("artifactory_requests_can_retrieve_remote_artifacts", false),
-		Repositories:          d.getList("repositories"),
-		Description:           d.getString("description", false),
-		Notes:                 d.getString("notes", false),
-		DefaultDeploymentRepo: getDefaultDeploymentRepo(d),
+		Repositories:                    d.getList("repositories"),
+		Description:                     d.getString("description", false),
```

Comment:
`gofmt` ?

Reply:
did run gofmt ... will check why the alignment is different.